shanoir-ng-front/src/app/dataset-acquisitions/dataset-acquisition/dataset-acquisition.component.html

```diff
@@ -149,6 +149,20 @@ <h2 class="header command-zone">Create dataset acquisition</h2>
 							</ng-template>
 						</span>
 					</li>
+                    <li *ngIf="mode != 'create' && datasetAcquisition.origin == 'SOURCE'">
+                        <label>This acquisition has been copied :</label>
+                        <span class="right-col">
+                            <div *ngFor="let entityId of copyEntityIds">
```

Comment:
why not ```let entityId of datasetAcquisition.copies``` directly ? Then coupled with ```*ngIf="datasetAcquisition.copies?.length > 0"``` instead of ```datasetAcquisition.origin == 'SOURCE'"``` you may have no use of the enum anymore (if it's not used elsewhere)